src/main/java/controller/CreateUserController.java

```diff
@@ -0,0 +1,33 @@
+package controller;
+
+import db.DataBase;
+import model.*;
+
+import java.util.Map;
+
+public class CreateUserController implements Controller {
+
+    private static final Path path = Path.of("/user/create");
+
+    @Override
+    public boolean matchHttpMethodAndPath(HttpRequest request) {
+        return request.isMatch(HttpMethod.POST, path);
+    }
+
+    @Override
+    public HttpResponse execute(HttpRequest request) {
+        User user = new User(
+                request.getBodyValue("userId"),
+                request.getBodyValue("password"),
+                request.getBodyValue("name"),
+                request.getBodyValue("email")
+        );
+        DataBase.addUser(user);
+
+        return HttpResponse.of(
```

Comment:
이런 형태의 팩터리 메서드를 추가해보면 어떨까요

```suggestion
        return HttpResponse.ofFound("/index.html");
```